.github/workflows

```diff
@@ -0,0 +1,25 @@
+name: Get repository stats
```

Comment:
Think this file should be named something like `.github/workflows/<name.yaml>`.

Reply:
ouch, yes...... :innocent: